src/redux/categories/categories.js

```diff
@@ -0,0 +1,15 @@
+const STATUS = 'CHECK_STATUS';
```

Comment:
- [ ] Please note that the value of each action type needs to follow the [ducks pattern](https://github.com/erikras/ducks-modular-redux), so it will be something like:       `const ADD_BOOK = bookstore/books/ADD_BOOK` and `const REMOVE_BOOK = bookstore/books/REMOVE_BOOK`. Kindly correct this, please.